src/App.css

```diff
@@ -1,5 +1,191 @@
+@import url('https://fonts.googleapis.com/css2?family=Montserrat:wght@300;400;600;700&family=Roboto+Slab:wght@300;400;700&display=swap');
+
 * {
   margin: 0;
   padding: 0;
   box-sizing: border-box;
 }
+
+body {
+  font-size: 1.3rem;
+  font-family: 'Montserrat', sans-serif;
+  background-color: #fafafa;
+  color: #121212;
+}
+
+/* #root {
+  border: 1px solid;
+  width: 60%;
+  margin: 10% auto;
+  background-color: #ffffff;
+} */
```

Comment:
- [ ] Please remove all the commented codes from your codebase in order to make your code clean. :+1: